Sources/Extensions/Foundation/DateExtensions.swift

```diff
@@ -833,6 +833,31 @@ public extension Date {
 		let componentValue = components.value(for: component)!
 		return abs(componentValue) <= value
 	}
+    
+    /// SwifterSwift: Random date between two dates.
+    ///
+    ///     Date.random()
+    ///     Date.random(since: Date())
+    ///     Date.random(to: Date())
+    ///
+    /// - Parameters:
+    ///   - startDate: minimum date (default is Date.distantPast)
+    ///   - endDate: maximum date (default is Date.distantFuture)
+    /// - Returns: random date between two dates.
+    public static func random(since startDate: Date = Date.distantPast,
```

Comment:
Hey @akuzminskyi, thanks for submitting this PR! 😃 
What do you think about changing the external parameter names? 
`since` to `from`
`to` to `upTo`